packages/atomic/src/components/commerce/atomic-commerce-search-box/atomic-commerce-search-box.new.stories.tsx

```diff
@@ -11,7 +11,7 @@ const {decorator, play} = wrapInCommerceInterface({skipFirstSearch: true});
 
 const meta: Meta = {
   component: 'atomic-commerce-search-box',
-  title: 'Atomic-Commerce/Searchbox',
+  title: 'Atomic-Commerce/Interface Components/atomic-commerce-search-box',
```

Comment:
Why this name change ?

Reply:
Follows the nomenclature in the "Storybook Commerce Hierarchy" document shared by @louis-bompart